lib/iris/tests/test_pickling.py

```diff
@@ -48,19 +52,22 @@ def pickle_then_unpickle(self, obj):
             yield protocol, reconstructed_obj
 
     def assertCubeData(self, cube1, cube2):
-        np.testing.assert_array_equal(cube1.data, cube2.data)
+        self.assertArrayEqual(cube1.data, cube2.data)
 
-    @tests.skip_biggus
     @tests.skip_data
     def test_cube_pickle(self):
-        cube = iris.load_cube(tests.get_data_path(('PP', 'globClim1', 'theta.pp')))
+        cube = iris.load_cube(tests.get_data_path(('PP',
+                                                   'globClim1',
+                                                   'theta.pp')))
         self.assertTrue(cube.has_lazy_data())
-        self.assertCML(cube, ('cube_io', 'pickling', 'theta.cml'), checksum=False)
+        self.assertCML(cube, ('cube_io', 'pickling', 'theta.cml'),
+                       checksum=False)
 
-        for _, recon_cube in self.pickle_then_unpickle(cube):
+        for p, recon_cube in self.pickle_then_unpickle(cube):
             self.assertTrue(recon_cube.has_lazy_data())
-            self.assertCML(recon_cube, ('cube_io', 'pickling', 'theta.cml'), checksum=False)
-            self.assertCubeData(cube, recon_cube)
+            self.assertCML(recon_cube, ('cube_io', 'pickling', 'theta.cml'),
+                           checksum=False)
+            self.assertCubeData(cube.copy(), recon_cube)
```

Comment:
>  turns out `assertCubeData` used to be defined in a way which didn't realise the cube's data

Oh excellent. Do you think it would be more appropriate to revert `assertCubeData` to how it was before rather than copying the cube here?